### PR TITLE
chore: 💄 Improve site experience and fix #26528

### LINKED ISSUE #26510: api 适配屏幕较小的电脑
https://ant.design/components/config-provider-cn/
image

### LINKED ISSUE #26528: Modal close will roll up
https://github.com/ant-design/ant-design/pull/26507#pullrequestreview-479537379

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close #26528
close #26510

### 💡 Background and solution

- [x] remove docs site horizontal scrollbar 
- [x] improve API table style in small screen 
- [x] add debug link color to toc
- [x] https://github.com/ant-design/ant-design/pull/26538/commits/d661a73fe672b74572719131a2db61dada725aa8 fix https://github.com/ant-design/ant-design/issues/26528

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Fix Modal closing cause scroll position jump up.  |
| 🇨🇳 Chinese |   修复 Modal 关闭时导致滚动条向上跳动的问题。   |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by hengkx** [comment]
modal的问题 没有 解决 依然那样

**Comment by hengkx** [comment]
bug

**Comment by afc163** [comment]
解决的是 https://github.com/ant-design/ant-design/issues/26528#issuecomment-685333836 这个问题。

**Comment by afc163** [comment]
和宽度没关系，全屏也会出问题。

**Comment by shaodahong** [comment]
> ~解决的是 [#26528 (comment)](https://github.com/ant-design/ant-design/issues/26528#issuecomment-685333836) 这个问题。~
> 
> 确实还有问题。

这个是 focus tatget 的问题，和 site 没关系

**Comment by shaodahong** [comment]
如果想要不 focus 可以加上 `focusTriggerAfterClose: false`

**Comment by afc163** [comment]
focus 的按钮在视窗可见范围为啥会引起滚动条跳动？

**Comment by afc163** [comment]
可能和触发时机有关系，试试把 focusTriggerAfterClose 的行为放到滚动条重置之后再进行。

**Comment by shaodahong** [comment]
嗯，其实最好的属性应该是 `scrollIntoViewIfNeeded `，就是兼容性太差

**Comment by shaodahong** [comment]
用 `setTimeout` 试了下不行，300 ms 了，感觉可以做个兼容处理，supported `scrollIntoViewIfNeeded` 然后做降级

**Comment by afc163** [comment]
和 scrollIntoViewIfNeeded 没关系，这是调用 focus，不需要 scrollIntoView。

File: package.json [hunk 1]
```diff
@@ -121,7 +121,7 @@
     "rc-cascader": "~1.3.0",
     "rc-checkbox": "~2.3.0",
     "rc-collapse": "~2.0.0",
-    "rc-dialog": "~8.2.0",
+    "rc-dialog": "~8.2.1",
```
Comment (afc163) [inline package.json:124]: https://github.com/react-component/dialog/pull/202

**Comment by afc163** [comment]
https://preview-26538-ant-design.surge.sh 实测没问题了。

**Pull request merged**